tools/accuracy_checker/requirements.in

```diff
@@ -1,36 +1,37 @@
 # progress bar
-tqdm
+tqdm~=4.31.1
```

Comment:
why this is strict?

Reply:
as POT dependency, see presentation option 1

Reply:
I am not think that POT requirements should be preliminary on AC own (not all AC users use POT) and moreover it is old version of package (so at least it should be updated on 4.60 in POT)
cc @AlexKoff88 @MaximProshin

Reply:
We will try to align it with NNCF  as well, it will be like this tqdm>=4.54.1 Are you ok?

Reply:
yes, it is ok for me